packages/core/src/components/Tabs/Tab.jsx

```diff
@@ -1,8 +1,9 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+// eslint-disable-next-line sort-imports, no-unused-vars
```

Comment:
In this case, ESLint throwing an error is good, and we can re-order the imports to fix the error. For the `h` import, is this necessary since it's not being used? If this is for Preact, could the solution be what's mentioned in [this issue](https://github.com/developit/preact/issues/1010)?

Reply:
From what I remember, adding `h` prevented it from crashing, so it is required. And the ESLint line is what I had to do because re-ordering didn't help.